### PR TITLE
Add a few components

### PR DESCRIPTION
**Pull request by cbabnik**
This adds 4 components
`utility`, `help`, `audio`, and `pictures`.

utility has the commands `roll` `random` `coinflip` and `math`
audio has the command `play`
pictures has the commands `burger` and `addBurger`
help has various help messages

could honestly be a cleaner pull request. utility is tested but audio and pictures are not.

### CODE REVIEW

File: components/utility.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        this.addCommand("-random (\\S+(?:,[^\\s,]+)+)$", this.random);
+        this.addCommand("-math", this.calculateInfo);
+        this.addCommand("-math ([-\\+\\*\\/\\.\\d\\(\\)]*)$", this.calculate);
+        this.addCommand('-coinflip$', () => this.coinflip("", ""));
+        this.addCommand('-coinflip (\\S+) (\\S+)$', this.coinflip);
+        this.addCommand('-coinflip "(.*)" "(.*)"$', this.coinflip);
+    }
+
+    rollInfo(metaInfo) {
+        if (metaInfo.commandMatchesCount === 1)
```
Comment (KHartwig) [inline components/utility.js]: Add body brackets to multi-line if/else

File: components/audio.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+const ID = "audio";
+
+class Audio extends Component {
+    constructor() {
+        super(ID);
+        this.addCommand("-play (.*)", this.playAudio);
+    }
+
+    playAudio(fileName) {
```
Comment (KHartwig) [inline components/audio.js]: Maybe want to be able to omit the file extension in the fileName when parsing, could be a later improvement

File: components/help.js [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+        this.addCommand("\\?help math", this.mathHelp);
+        this.addCommand("\\?coinflip", this.coinflipHelp);
+        this.addCommand("-help coinflip", this.coinflipHelp);
+        this.addCommand("\\?help coinflip", this.coinflipHelp);
+        this.addCommand("\\?burger", this.burgerHelp);
+        this.addCommand("-help burger", this.burgerHelp);
+        this.addCommand("\\?help burger", this.burgerHelp);
+        this.addCommand("\\?add[bB]urger", this.addBurgerHelp);
+        this.addCommand("-help add[bB]urger", this.addBurgerHelp);
+        this.addCommand("\\?help add[bB]urger", this.addBurgerHelp);
```
Comment (KHartwig) [inline components/help.js]: Can use javascript regex literals (ie /\?help add[bB]urger/ )

File: components/audio.js (hunk 2 above)
Comment (cbabnik) [reply components/audio.js]: Ill add it to the issue backlog

**Pull request closed without merging**